ui/lib/config-ui/addon/components/messages/tab-page-header.hbs

```diff
@@ -10,43 +10,45 @@
     </p.top>
   {{/if}}
   <p.levelLeft>
-    <Hds::Text::Display @tag="h2" @size="500" class="has-top-margin-xl">
-      Custom messages
+    <Hds::Text::Display @tag="h2" @size="500" class="has-top-margin-m" data-test-page-title>
+      {{@pageTitle}}
     </Hds::Text::Display>
   </p.levelLeft>
 </PageHeader>
 
-<div class="tabs-container box is-bottomless is-marginless is-paddingless">
-  <nav class="tabs" aria-label="custom-messages">
-    <ul>
-      {{! Explicitly setting page to 1 here since we want to reset the page param on transition}}
-      <LinkTo
-        class={{if @authenticated "active"}}
-        @route="messages"
-        @query={{hash authenticated=true page=1}}
-        data-test-custom-messages-tab="After user logs in"
-      >
-        After user logs in
-      </LinkTo>
-      <LinkTo
-        class={{unless @authenticated "active"}}
-        @route="messages"
-        @query={{hash authenticated=false page=1}}
-        data-test-custom-messages-tab="On login page"
-      >
-        On login page
-      </LinkTo>
-    </ul>
-  </nav>
-</div>
+{{#if @showTabs}}
+  <div class="tabs-container box is-bottomless is-marginless is-paddingless">
+    <nav class="tabs" aria-label="custom-messages">
+      <ul>
+        {{! Explicitly setting page to 1 here since we want to reset the page param on transition}}
+        <LinkTo
+          class={{if @authenticated "active"}}
+          @route="messages"
+          @query={{hash authenticated=true page=1}}
+          data-test-custom-messages-tab="After user logs in"
+        >
+          After user logs in
+        </LinkTo>
+        <LinkTo
+          class={{unless @authenticated "active"}}
+          @route="messages"
+          @query={{hash authenticated=false page=1}}
+          data-test-custom-messages-tab="On login page"
+        >
+          On login page
+        </LinkTo>
+      </ul>
+    </nav>
+  </div>
 
-{{#if (or (has-block "toolbarFilters") (has-block "toolbarActions"))}}
-  <Toolbar>
-    <ToolbarFilters>
-      {{yield to="toolbarFilters"}}
-    </ToolbarFilters>
-    <ToolbarActions>
-      {{yield to="toolbarActions"}}
-    </ToolbarActions>
-  </Toolbar>
+  {{#if (or (has-block "toolbarFilters") (has-block "toolbarActions"))}}
+    <Toolbar>
```

Comment:
Another a11y thing -- please add `aria-label` to Toolbar, ToolbarFilters, and ToolbarActions describing what kind of interactivity is within